python/cugraph/cugraph/structure/renumber_wrapper.pyx

```diff
@@ -65,7 +65,8 @@ cdef renumber_helper(shuffled_vertices_t* ptr_maj_min_w, vertex_t, weights):
             move(pair_s_weights.first), weight_t, "shuffled_weights")
         if shuffled_weights_series is None:
             shuffled_df['value']= cudf.Series(dtype=weight_t)
-        shuffled_df['value']= shuffled_weights_series
+        else:
+            shuffled_df['value']= shuffled_weights_series
```

Comment:
I believe we should do the same for `shuffled_minor_series` and `shuffled_major_series`

Reply:
Good catch.  Wasn't causing an error, but the code is, in fact, incorrect.  Updated in the last push.